courseSchedule.java

```diff
@@ -0,0 +1,67 @@
+// 207.
+
+//time - O(n)
```

Comment:
Time Complexity:O(2⋅∣E∣+∣V∣) where ∣V∣ is the number of courses, and ∣E∣ is the number of dependencies. -Leetcode